### PR TITLE
renamed RecipientService to MediationRecipientService

### PR DESCRIPTION
**Pull request by burdettadam**
Signed-off-by: Adam Burdett <burdettadam@gmail.com>

### CODE REVIEW

File: src/modules/connections/ConnectionsModule.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { AgentConfig } from '../../agent/AgentConfig'
 import { Dispatcher } from '../../agent/Dispatcher'
 import { MessageSender } from '../../agent/MessageSender'
 import { createOutboundMessage } from '../../agent/helpers'
-import { RecipientService as MediationRecipientService } from '../routing/services/RecipientService'
+import { MediationRecipientService as MediationRecipientService } from '../routing/services/MediationRecipientService'
```
Comment (TimoGlastra) [inline src/modules/connections/ConnectionsModule.ts:10]:
```suggestion
import { MediationRecipientService } from '../routing/services/MediationRecipientService'
```

File: README.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Some features are not yet supported, but are on our roadmap. Check [the roadmap]
 - ✅ HTTP Transport
 - 🚧 Revocation of Indy Credentials
 - 🚧 Electron
-- 🚧 Mediator Coordination Protocol ([RFC 0211](https://github.com/hyperledger/aries-rfcs/blob/master/features/0211-route-coordination/README.md))
+- ✅ Mediator Coordination Protocol ([RFC 0211](https://github.com/hyperledger/aries-rfcs/blob/master/features/0211-route-coordination/README.md))
```
Comment (TimoGlastra) [inline README.md:68]: Can you move it above the under construction items "🚧"?

**Review by TimoGlastra — APPROVED** [review]
Few small comments

File: README.md (hunk 2 above)
Comment (burdettadam) [reply README.md]: moved.

**Comment by TimoGlastra** [comment]
Non controversial change based on feedback in mediator pr. Merging

**Pull request merged**